src/view/com/modals/SelfLabel.tsx

```diff
@@ -116,23 +119,26 @@ export const Component = observer(function Component({
 
               <Text style={[pal.text, styles.adultExplainer]}>
                 {selected.includes('sexual') ? (
-                  <>Pictures meant for adults.</>
+                  <Trans>Pictures meant for adults.</Trans>
                 ) : selected.includes('nudity') ? (
-                  <>Artistic or non-erotic nudity.</>
+                  <Trans>Artistic or non-erotic nudity.</Trans>
                 ) : selected.includes('porn') ? (
-                  <>Sexual activity or erotic nudity.</>
+                  <Trans>Sexual activity or erotic nudity.</Trans>
                 ) : (
-                  <>If none are selected, suitable for all ages.</>
+                  <Trans>If none are selected, suitable for all ages.</Trans>
                 )}
               </Text>
             </>
           ) : (
             <View>
               <Text style={[pal.textLight]}>
                 <Text type="md-bold" style={[pal.textLight]}>
-                  Not Applicable
+                  <Trans>Not Applicable</Trans>
                 </Text>
-                . This warning is only available for posts with media attached.
+                <Trans>
+                  . This warning is only available for posts with media
```

Comment:
Let's co-locate the `.` with `Not Applicable`

Reply:
Done